testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -43,7 +43,9 @@
         <activity
             android:name="com.microsoft.identity.client.testapp.MainActivity"
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="stateHidden"
+            android:taskAffinity=""
+            >
```

Comment:
```suggestion
            android:taskAffinity="">
```

Reply:
I'm actually going to remove this.  We need to modify this for different automation runs and manual test passes.